### PR TITLE
Release flink 1.15.1

### PR DESCRIPTION
**Pull request by alpinegizmo**
Preparing for the 1.15.1 release.

### CODE REVIEW

File: _posts/2022-06-29-release-1.15.1.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+layout: post
+title:  "Apache Flink 1.15.1 Release Announcement"
+date: 2022-06-29T08:00:00.000Z
+categories: news
+authors:
+- knaufk:
+  name: "David Anderson"
+  twitter: "alpinegizmo"
+
+excerpt: The Apache Flink Community is please to announce a bug fix release for Flink 1.15.
```
Comment (HuangXingBo) [inline _posts/2022-06-29-release-1.15.1.md]:
```suggestion
excerpt: The Apache Flink Community is pleased to announce a bug fix release for Flink 1.15.
```

**Review by HuangXingBo — APPROVED** [review]

File: _config.yml [hunk 2]
```diff
@@ -283,6 +283,10 @@ component_releases:
 release_archive:
     flink:
       - version_short: "1.15"
+        version_long: 1.15.1
+        release_date: 2022-06-29
```
Comment (knaufk) [inline _config.yml]: reminder to update

File: _posts/2022-06-29-release-1.15.1.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27683'>FLINK-27683</a>] -         Insert into (column1, column2) Values(.....) fails with SQL hints
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27711'>FLINK-27711</a>] -         Correct the typo of set_topics_pattern by changing it to set_topic_pattern for Pulsar Connector
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27733'>FLINK-27733</a>] -         Rework on_timer output behind watermark bug fix
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27734'>FLINK-27734</a>] -         Not showing checkpoint interval properly in WebUI when checkpoint is disabled
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27751'>FLINK-27751</a>] -         Dependency resolution from repository.jboss.org fails on CI
```
Comment (knaufk) [inline _posts/2022-06-29-release-1.15.1.md]: nit: exclude because its CI

File: _posts/2022-06-29-release-1.15.1.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27368'>FLINK-27368</a>] -         SQL CAST(&#39; 1 &#39; as BIGINT) returns wrong result
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27409'>FLINK-27409</a>] -         Cleanup stale slot allocation record when the resource requirement of a job is empty
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27418'>FLINK-27418</a>] -         Flink SQL TopN result is wrong
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27420'>FLINK-27420</a>] -         Suspended SlotManager fails to re-register metrics when started again
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27442'>FLINK-27442</a>] -         Module flink-sql-avro-confluent-registry does not configure Confluent repo
```
Comment (knaufk) [inline _posts/2022-06-29-release-1.15.1.md]: nit: exclude because its build-system

File: _posts/2022-06-29-release-1.15.1.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title:  "Apache Flink 1.15.1 Release Announcement"
+date: 2022-06-29T08:00:00.000Z
```
Comment (knaufk) [inline _posts/2022-06-29-release-1.15.1.md]: reminder to update

**Review by knaufk — APPROVED** [review]
LGTM. Just two nits

**Pull request merged**